sklearn/cluster/bicluster.py

```diff
@@ -228,15 +228,18 @@ class SpectralCoclustering(BaseSpectral):
         chosen and the algorithm runs once. Otherwise, the algorithm
         is run for each initialization and the best solution chosen.
 
-    n_jobs : int, optional, default: 1
+    n_jobs : int or None, optional (default=None)
         The number of jobs to use for the computation. This works by breaking
         down the pairwise matrix into n_jobs even slices and computing them in
         parallel.
 
+        If ``None``, then the number of jobs is set to 1 unless current joblib
```

Comment:
Maybe remove "then" and possibly "unless the current"

Reply:
I meant: possibly add "the" in "unless current"..

Reply:
@rth So you mean something like: If ``None``, the number of jobs is set to 1 unless the current joblib backend context specifies otherwise?

Reply:
yes, but let's wait for @jnothman 's opinion before propagating these changes everywhere..